packages/react/src/utilities/ThemeProvider/renderThemeProvider.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react';
-import { CustomizerContext, getNativeElementProps, omit } from '@fluentui/utilities';
+import { CustomizerContext, FocusRectsProvider, getNativeElementProps, omit } from '@fluentui/utilities';
 import { ThemeContext } from './ThemeContext';
 import type { ThemeProviderState } from './ThemeProvider.types';
 
 export const renderThemeProvider = (state: ThemeProviderState) => {
-  const { theme, customizerContext } = state;
+  const { customizerContext, focusRectsContext, theme } = state;
   const Root = state.as || 'div';
   const rootProps = typeof state.as === 'string' ? getNativeElementProps(state.as, state) : omit(state, ['as']);
 
   return (
     <ThemeContext.Provider value={theme}>
       <CustomizerContext.Provider value={customizerContext}>
-        <Root {...rootProps} />
+        <FocusRectsProvider value={focusRectsContext}>
+          <div>
+            <Root {...rootProps} />
+          </div>
```

Comment:
Is this extra div needed?

Reply:
Not needed, it is a remnant from when I was testing things, removing it 😄